pkg/probes/curl/userdata-template.yaml

```diff
@@ -1,12 +1,12 @@
 #cloud-config
-# network-verifier-required-variables=TIMEOUT,DELAY,URLS
+# network-verifier-required-variables=CURL_COMMAND,DELAY
 ${CACERT_RENDERED}
 runcmd:
   - systemctl mask --now serial-getty@ttyS0.service
   - dmesg -D
   - echo "${USERDATA_BEGIN}" >/dev/ttyS0
   - export http_proxy=${HTTP_PROXY} https_proxy=${HTTPS_PROXY} no_proxy="${NO_PROXY}"
-  - curl --capath /etc/pki/tls/certs/ --proxy-capath /etc/pki/tls/certs/ --retry 3 --retry-connrefused -t B -Z -s -I -m ${TIMEOUT} -w "%{stderr}${LINE_PREFIX}%{json}\n" ${CURLOPT} ${URLS} --proto =http,https,telnet ${TLSDISABLED_URLS_RENDERED} 2>/dev/ttyS0
+  - ${CURL_COMMAND} 2>/dev/ttyS0
```

Comment:
Should this be quoted to prevent shell expansion? Unsure if cloud-init does that for us or not, but worth confirming/testing.

Reply:
If I do this. All the curl_json_tests.go unit tests fail.

Reply:
Should `${CURL_COMMAND}` be quoted? Don't we _want_ shell expansion here? Line 7 quotes the variable but that's because it's an argument to echo. In Line 9's case, we replaced a command and its arguments with a variable. If we quote that variable, none of the arguments will expand. Here's my test in bash.

```
bash-5.2$ CURL_COMMAND='curl -sSLi'
bash-5.2$ "${CURL_COMMAND}" https://www.redhat.com
bash: curl -sSLi: command not found
```

Dropping the quotes runs the curl as expected.